blocks/related-articles/related-articles.js

```diff
@@ -8,6 +8,7 @@ function getCategoryForReadMore() {
 }
 
 export default async function decorateBlock(block) {
+  loadCSS(`${window.hlx.codeBasePath}/blocks/article-list/article-list.css`);
```

Comment:
Move this to article-list.js.

Reply:
what's "this" @mhaack?
thing is: if the related articles are loaded, the css is missing. so what is it I move to article-list.js?
sorry, I may just not get what you mean :)

Reply:
'related articles' has specific css, but it builds on the article list

Reply:
Yes hence `article-list.css` should only be loaded by `article-list.js` not the blocks consuming it.

Reply:
ok, now after couple hours' sleep I see what you mean. done.

Reply:
:-) LGTM now